.bldr.toml

```diff
@@ -417,6 +417,8 @@ plan_path = "gperftools"
 plan_path = "gpgme"
 [gradle]
 plan_path = "gradle"
+[gradle4]
+plan_path = "gradle4"
```

Comment:
Since gradle4 sources the gradle plan, it should also be triggered on changes from the gradle path.